solc/jsonCompiler.cpp

```diff
@@ -128,6 +128,11 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 	input["settings"]["optimizer"]["enabled"] = _optimize;
 	input["settings"]["optimizer"]["runs"] = 200;
 
+	// Enable all SourceUnit-level outputs.
+	input["settings"]["outputSelection"]["*"][""][0] = "*";
```

Comment:
Does `[0]` automatically make it an array?

Reply:
Yep. Had an ugly large code before to do it manually.